changelogs/unreleased/4201-stevesloka-minor.md

```diff
@@ -0,0 +1,5 @@
+### HTTP Request Redirect Policy 
```

Comment:
As part of making `service` optional, this needs an update.

Something like:

### HTTPProxy.Route.Service field now optional

As part of implementing Request Redirect Policy, HTTPProxy.Route.Service has been made optional. Either the Service or HTTPRequestRedirectPolicy fields must be set under the HTTPProxy.Route for the HTTPProxy to be valid. Not having at least one of these fields set will mean that the Route is rejected, and the appropriate Condition set on the proxy.